src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksVendorAchDataProcessingIntoKfsServiceImpl.java

```diff
@@ -102,23 +102,12 @@ private boolean kfsPayeeAchMaintenanceDocumentValidated(MaintenanceDocument paat
         return documentValidated;
     }
 
-    private boolean kfsPayeeAchMaintenceDocumentRouted(MaintenanceDocument paatMaintenceDoc, PaymentWorksNewVendorPayeeAchBatchReportData reportData, PaymentWorksVendor pmwVendor) {
-        boolean documentRouted = false;
-        try {
-            String annotationMessage = PaymentWorksConstants.KFSPayeeAchMaintenanceDocumentConstants.PAYMENTWORKS_NEW_VENDOR_ACH_CREATE_ROUTE_ANNOTATION + 
-                    pmwVendor.getPmwVendorRequestId();
-            getDocumentService().routeDocument(paatMaintenceDoc, annotationMessage, null);
-            LOG.info("kfsPayeeAchMaintenceDocumentRouted: paatMaintenceDoc routed.");
-            documentRouted = true;
-        } catch (WorkflowException we) {
-            List<String> edocCreateErrors = getPaymentWorksBatchUtilityService().convertReportDataValidationErrors(GlobalVariables.getMessageMap().getErrorMessages());
-            captureKfsPayeeAchProcessingErrorsForVendor(pmwVendor, reportData, edocCreateErrors);
-            LOG.error("kfsPayeeAchMaintenceDocumentRouted: eDoc routing error(s): " + edocCreateErrors.toString());
-            LOG.error("kfsPayeeAchMaintenceDocumentRouted: eDoc routing exception caught: " + we.getMessage());
-        } finally {
-            GlobalVariables.getMessageMap().clearErrorMessages();
-        }
-        return documentRouted;
+    private void kfsPayeeAchMaintenceDocumentRouted(MaintenanceDocument paatMaintenceDoc, PaymentWorksNewVendorPayeeAchBatchReportData reportData, PaymentWorksVendor pmwVendor) {
```

Comment:
If it's no longer necessary for this method to return a boolean, it would be a good idea to adjust its name accordingly. At the moment its name implies that it returns a boolean value.